### DISCUSSION TITLE
After 1.4 update pwas does not work

### DISCUSSION THREAD

**Discussion by panagiotisevaggelou** [filips123/PWAsForFirefox#122] [Q&A]
After I updated to the latest version it suddenly stopped all my pwas from working. Now I install a site like Youtube as a pwa but it doesn't let me launce it, please help.

**Comment by filips123** [comment]
Can you describe how you tried to launch PWA and what exactly happend? Also, please provide info about your system:

* Operating system:
* System architecture: (64-bit/32-bit/...)
* Desktop environment: (If on Linux)
* Installation method: (MSI/DEB/RPM/AUR/source...)
* PWAsForFirefox extension version:
* PWAsForFirefox native version:
* Firefox version:

Then, please check [troubleshooting steps](https://github.com/filips123/PWAsForFirefox/wiki/Troubleshooting), and, if your error is not fixed, provide me log files as described in that page.

**Comment by panagiotisevaggelou** [comment]
OS: Windows 10 Home 20H2
System architecture: 64-bit
Installation method: MSI
PWAsForFirefox extension version: 1.4.0
PWAsForFirefox native version: 1.4.0
Firefox version: 93.0 (64-bit)

I had already installed the program and using it normally, having some pwas pined to my taskbar.
Some days after I updated the program to version 1.4.0, suddenly I was clicking on the pwa icon on taskbar and it just refused to launch.
I also tried launching the pwa from the app list in start menu and again nothing would happen.
If I tried to launch the pwa from the addon page it would launch but if I pinned it to the taskbar, close it and then launch it again from the taskbar It wouldn't launch.
Some other pwas wouldn't even launch at all, but just display a 'new tab' page, not the url of the pwa.
I tried:
- deleting and reinstalling the addon
- I had installed firefox version 96.0.2 and I downgraded to 93.0
- i pressed the button to reinstall timeline
- I pressed the button to update web apps
Nothing worked.
I will try the troubleshooting steps also, thanks.

**Reply by filips123** [reply]
If you still haven't fixed the issue, please provide me log files, as unfortunately I cannot know why it happened without them.
